docs/latest/extensions/modules/sop-class-handler.md

```diff
@@ -15,13 +15,13 @@ A `displaySet` has the following shape:
 ```js
 return {
   plugin: 'html',
-  modality: 'SR',
+  Modality: 'SR',
   displaySetInstanceUid: 0,
   wadoRoot: study.getData().wadoRoot,
   wadoUri: instance.getData().wadouri,
-  sopInstanceUid: instance.getSOPInstanceUID(),
-  seriesInstanceUid: series.getSeriesInstanceUID(),
-  studyInstanceUid: study.getStudyInstanceUID(),
+  SOPInstanceUID: instance.getSOPInstanceUID(),
+  SeriesInstanceUID: series.getSeriesInstanceUID(),
+  StudyInstanceUID: study.getStudyInstanceUID(),
```

Comment:
ESLint list? That'd be a lot of words to auto-check

Reply:
This is a good idea, we should think about it in another issue though.